src/Sql/Sql/Replication/Cmdlet/GetAzureSqlDatabaseReplicationLink.cs

```diff
@@ -73,7 +73,7 @@ protected override IEnumerable<AzureReplicationLinkModel> GetEntity()
         {
             ICollection<AzureReplicationLinkModel> results;
 
-            if (MyInvocation.BoundParameters.ContainsKey(PartnerServerName) && !WildcardPattern.ContainsWildcardCharacters(PartnerServerName))
+            if (MyInvocation.BoundParameters.ContainsKey("PartnerServerName") && !WildcardPattern.ContainsWildcardCharacters(PartnerServerName))
```

Comment:
Suggest use `nameof` to avoid magic string
```suggestion
            if (MyInvocation.BoundParameters.ContainsKey(nameof(PartnerServerName)) && !WildcardPattern.ContainsWildcardCharacters(PartnerServerName))
```